app/views/software_choices_filter.scala.html

```diff
@@ -83,6 +83,7 @@ <h1 class="govuk-heading-xl">@messages("softwareChoices.filter.title")</h1>
 
```

Comment:
The page title for error_template needs updating to have the correct suffix as per point 4 on the story.